pilot/pkg/model/config_test.go

```diff
@@ -148,24 +148,24 @@ func TestSubsetKey(t *testing.T) {
 			hostname: "hostname",
 			subset:   "subset",
 			port:     80,
-			want:     "outbound|80|subset|hostname",
+			want:     "outbound|80|subset|hostname|" + string(model.TunnelNone),
```

Comment:
Why do we use `model.TunnelNone` instead of just empty string like we do for missing subset?

Reply:
I reverted to 4 or 5 parts: tunnel-none will build the original cluster name.
Var length is not ideal, but build <-> parse is 1:1 mapping